src/apps/loyalty/analytics.ts

```diff
@@ -0,0 +1,65 @@
+/*
+This file contains common/helper analytics methods that should
+only be used client-side. For example, calling `initAnalytics()`
+will set up all the standard pageview/user/bounce rate tracking.
+
+This relies on `window.analytics` (wrapping Segment's Analytics.js)
+to perform its analytics calls. That library automatically includes
+enriched client-side information in its payload, which is why we
+require/prefer that some calls happen cient-side.
+*/
+import { extend, pick } from "lodash"
+import * as sharify from "sharify"
+
+import { LoginResponseLocalData } from "./types"
+const data = sharify.data as LoginResponseLocalData
+
+export function initAnalytics() {
+  if (typeof(window) === "undefined") {
+    throw new Error("Method should only be used client-side")
+  }
```

Comment:
nice call, if this repo ends up with the analytics changes from https://github.com/artsy/emission/pull/491 - this could be turned into a function annotation.